minimesos/src/main/java/com/containersol/minimesos/mesos/ClusterArchitecture.java

```diff
@@ -182,6 +185,7 @@ public Builder withZooKeeper(ZooKeeperConfig zooKeeperConfig) {
          * Provide a custom implementation of the {@link ZooKeeper} container.
          *
          * @param zooKeeper must extend from {@link ZooKeeper}
+         * @return TODO(mwl)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1135)